odoo/addons/test_assetsbundle/tests/test_js_transpiler.py

```diff
@@ -209,20 +209,20 @@ def test_05_functions(self):
         expected_result = """odoo.define('@test_assetsbundle/functions', async function (require) {
 'use strict';
 let __exports = {};
-const sayHello = __exports.sayHello = function sayHello() {
+__exports.sayHello = sayHello; function sayHello() {
```

Comment:
why not just this: `__exports.sayHello = function sayHello() {` ?

Reply:
You can't call "sayHello" in the file with your implementation.